Swift/ReplicatorConfiguration.swift

```diff
@@ -476,5 +498,11 @@ public struct ReplicatorConfiguration {
     }
     
     var collectionConfigs = [Collection: CollectionConfiguration]()
+    
+    // [collection.fullName : collection]
+    var collectionMap = [String: Collection]()
```

Comment:
I forgot the rule of Swift's dictionary key. Is collectionConfigs alone not working?

Reply:
we don't have unique collection objects for the given name/scopeName. Every time a new object will be created so comparing with object might not work.

Reply:
I think a proper way to implement this is to have Collection class implements Hashable.

Reply:
ok.

Reply:
Something is not right, even though collection was hashable, last time it showed some error. Which is not happening right now. Let me remove this and try again.